packages/cli/src/util/env/env-target.ts

```diff
@@ -1,22 +1,30 @@
-import { ProjectEnvTarget } from '@vercel-internals/types';
+import type { ProjectEnvTarget } from '@vercel-internals/types';
 
-function envTargets(): string[] {
-  return Object.values(ProjectEnvTarget);
-}
-
-export function getEnvTargetChoices() {
-  return Object.entries(ProjectEnvTarget).map(([key, value]) => ({
-    name: key,
-    value: value,
-  }));
-}
+export const envTargetChoices = [
+  {
+    name: 'Production',
+    value: 'production',
+  },
+  {
+    name: 'Preview',
+    value: 'preview',
+  },
+  {
+    name: 'Development',
+    value: 'development',
+  },
+] as const;
```

Comment:
Can we avoid duplicating code by using the pattern here:

https://twitter.com/styfle/status/1208787362613989381

I would expect a single source of truth.

Reply:
So we do wanna use constants for these? Sorry I misunderstood what you were asking in the slack thread